src/main/java/org/sciborgs1155/robot/drive/SimModule.java

```diff
@@ -59,4 +82,83 @@ public void resetEncoders() {
 
   @Override
   public void close() {}
+
+  @Override
+  public String getName() {
+    return name;
+  }
+
+  @Override
+  public SwerveModuleState state() {
+    return new SwerveModuleState(driveVelocity(), rotation());
+  }
+
+  @Override
+  public SwerveModulePosition position() {
+    return new SwerveModulePosition(drivePosition(), rotation());
+  }
+
+  @Override
+  public SwerveModuleState desiredState() {
+    return setpoint;
+  }
+
+  @Override
+  public void setDriveSetpoint(double velocity) {}
+
+  @Override
+  public void setTurnSetpoint(double angle) {}
+
+  @Override
+  public void updateSetpoint(SwerveModuleState setpoint, ControlMode mode) {
```

Comment:
Sim should follow the same logic as the rest of the modules, and we both know that rev's onboarding pid code could never be this complex